include/abb_librws/v1_0/rws.h

```diff
@@ -391,6 +391,11 @@ namespace abb :: rws :: v1_0
      */
     static const std::string LOGOUT;
 
+    /**
+     * \brief Progress.
+     */
+    static const std::string PROGRESS;
+
```

Comment:
I think defining these constants in a header is an overkill, and it exposes an implementation detail.